source/octf/proto/defs.proto

```diff
@@ -70,6 +70,23 @@ message LogTrace {
     string trace = 4;
 }
 
+message EventIoTimestamp {
```

Comment:
Just Timestamp

Reply:
Maybe we can use proto already existing type: https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/timestamp.proto?